node/resolvers/brand/upload.ts

```diff
@@ -0,0 +1,155 @@
+import { ReadStream } from 'fs'
+import { VBase } from '@vtex/api'
+
+import {
+  CatalogGQL
+} from '../../clients/catalogGQL'
+import {
+  pacer,
+  BRAND_NAME,
+  calculateExportProcessTime,
+  BRAND_TRANSLATION_UPLOAD,
+  calculateBreakpoints,
+  CALLS_PER_MINUTE,
+  parseStreamToJSON
+} from '../../utils'
+
+const uploadBrandAsync = async (
```

Comment:
This function is almost the same as the one we have for products, only changing  the bucket name where it's saved on VBASE. Do you minding making it reusable for both process, please? I think it will be useful for the other entries we will need to create.

Reply:
done